spa/src/components/Chat.vue

```diff
@@ -195,7 +195,7 @@ export default Vue.extend({
       });
       this.$socket.on("AV:del", event => {
         this.systemMessage(event.username + " has left.");
-        this.users = this.users.filter((u) => u.username !== event.username);
+        this.users = this.users.filter((u) => u.id !== event.id);
```

Comment:
We have user database IDs exposed to the client? That seems like a big security smell.

Reply:
These should be socket.io autogenerated socket IDs, not database IDs.

Reply:
Oh awesome, thanks for clearing that up!